test/unit/ajax.js

```diff
@@ -1590,6 +1592,19 @@ module( "ajax", {
 				ok( false, "error" );
 			}
 		} );
+
+		if ( jQuery.support.cors ) {
+			ajaxTest( "gh-1684 - jQuery.ajax() - support cross-domain PATCH in IE10-11", 1, {
+				url: "http://httpbin.org/patch",
```

Comment:
> Also, a detection via if ( window.ActiveXObject ) { ... } fails in IE11 because it was used for bad browser sniffs.

`window.ActiveXObject !== undefined` will succeed, though since we use it ourselves. ;) But this would be stepping over our implementation, `document.documentMode` seems safer here.